zookeeper-contrib/zookeeper-contrib-loggraph/src/main/java/org/apache/zookeeper/graph/RandomAccessFileReader.java

```diff
@@ -112,9 +112,7 @@ synchronized public int read(char[] cbuf, int off, int len) throws IOException {
     }
 
     synchronized public int read(byte[] buf, int off, int len) throws IOException {
-	if (LOG.isTraceEnabled()) {
-	    LOG.trace("read(buf, off=" + off + ", len=" + len);
-	}
+	    LOG.trace("read(buf, off={}, len={}" off, len);
 
```

Comment:
compile error?

Reply:
Woops.  That's embarrassing.  Must have accidentally hit a few key strokes there.  Thanks for pointing it out. I've update the PR to fix this.